validator/testdata/feature_tests/amp-date-countdown.html

```diff
@@ -0,0 +1,164 @@
+<!--
```

Comment:
It looks like this test is missing a matching .out file. Please create one, and see the documentation at https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md#test-output-files for how to run the test.

Reply:
I still don't see the out file -- perhaps you forgot to `git add`?

Reply:
To update tests you'll need to run `gulp validator --update_tests`. There may currently be an issue with it and it's being fixed in #15494

Reply:
Oh, actually, I see that there's a test file in both here (validator/testdata/feature_tests/amp-date-countdown.html) and in extensions/amp-date-countdown/0.1/test/validator-amp-date-countdown.html - and that the latter has a matching .out file, as desired.

@honeybadgerdontcare Can you confirm that extensions/amp-date-countdown/0.1/test/ is the proper location for this test?

@leonalicious If @honeybadgerdontcare confirms, could you delete this copy of the file? Thanks!

Reply:
yes, please delete this file and keep the one in the extension test directory.

Reply:
i cant run `gulp validator --update_tests` stuck with `Protobuf compiler not found.` and i did try on the guide on https://medium.com/@erika_dike/installing-the-protobuf-compiler-on-a-mac-a0d397af46b8, seems it is still not working

Reply:
Hm... can you run `protoc` on the commandline? The binary needs to be in your $PATH. i.e the `make install` step is necessary.

If that didn't work, here's another set of instructions you could try: http://macappstore.org/protobuf/ -- I've heard that the Homebrew packages are generally well-regarded.

Reply:
thank you, i am done with the installing already ~ but i am encountering another error

![image](https://user-images.githubusercontent.com/4065175/40468845-2940de58-5f61-11e8-8be8-d5c197176a85.png)

Reply:
OK, I was able to patch in your change and run `gulp validator`. Two things:

1. The test at `extensions/amp-date-countdown/0.1/test/validator-amp-date-countdown.html` generated the following error: [error message](https://gist.github.com/twifkak/2da9721b7ce33993ea47ab3040271c8b)

Also, the test at `validator/testdata/feature_tests/amp-date-countdown.html` fails due to a missing out file.

I generated a fix for both of these, apply with `git am`: [patch file](https://gist.githubusercontent.com/twifkak/f078e79f1ef75b7d9a9875a285475526/raw/00ce6e0ea6b52d9e3010f2b35305ba29fb73888a/0001-Fix-amp-date-countdown-validator-rules-and-test.patch)

2. I had trouble applying the patch. I ran `curl https://github.com/ampproject/amphtml/pull/15453.patch >/tmp/15453.patch && git am /tmp/15453.patch` and it failed at [this commit](https://github.com/ampproject/amphtml/pull/15453/commits/687d6314f517a4474035efb38529f2417bedcdc4). I suspect you might need to `git rebase -i` and squash this commit with its parent to get this PR to apply cleanly -- I'm not sure though.

Reply:
okay, thank you for your prompt assistance, i will try it over the weekend~